toolkit/offchain/src/lib.rs

```diff
@@ -14,6 +14,7 @@ pub mod ogmios_mock;
 pub mod permissioned_candidates;
 /// Utilities for handling Plutus script data
 mod plutus_script;
+mod register;
```

Comment:
Please add a comment that `register` is about block producer (candiate) registration messages